examples/auth-component-no-auth/src/lib.rs

```diff
@@ -19,13 +19,13 @@ fn my_panic(_info: &core::panic::PanicInfo) -> ! {
     loop {}
 }
 
-bindings::export!(AuthComponent with_types_in bindings);
+miden::miden_generate!();
```

Comment:
*Nit:* Maybe we should rename the macro to just `generate!`, so that this can read as `miden::generate!()` - it's a minor aesthetic improvement, but a lot easier to remember and say out loud.

Reply:
Done.